cordova-lib/src/plugman/install.js

```diff
@@ -71,7 +71,8 @@ module.exports = function installPlugin(platform, project_dir, id, plugins_dir,
     project_dir = cordovaUtil.convertToRealPathSafe(project_dir);
     plugins_dir = cordovaUtil.convertToRealPathSafe(plugins_dir);
     options = options || {};
-    options.is_top_level = true;
+    if (options.is_top_level === undefined)
```

Comment:
I changed as you suggested.
Do you mean you want to set options.is_top_level = true if options.is_top_level is set to null ?
In fact, the is_top_level is a boolean property, so a null value for this property is a bug although null is falsy in Javascript.
When options.is_top_level is set to null for some reason, options.hasOwnProperty('is_top_level') is  true.
That means, the new condition (options.hasOwnProperty('is_top_level') === false) will not change options.is_top_level from the null value to true.
Just wanted clarify.